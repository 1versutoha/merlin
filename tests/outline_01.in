["tell","start"]
["tell","source","type t1 = A | B of string\n"]
["seek","before",{"line":10,"col":0}]
["tell","start"]
["tell","source","\n\ntype t2 = { e1 : bool; e2 : int; }\n\nlet a = 1\n\nmodule M = struct\n  let m1 = \"abc\"\nend\n"]
["tell","source",""]
["outline"]
