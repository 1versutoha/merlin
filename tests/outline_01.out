["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":1,"col":0},"marker":false}]
["return",{"cursor":{"line":3,"col":0},"marker":false}]
["return",{"cursor":{"line":3,"col":0},"marker":false}]
["return",{"cursor":{"line":3,"col":0},"marker":false}]
["return",{"cursor":{"line":11,"col":0},"marker":false}]
["return",[{"name":"m1","kind":"Value","pos":{"line":9,"col":6},"children":[]},{"name":"a","kind":"Value","pos":{"line":6,"col":4},"children":[]},{"name":"t2","kind":"Type","pos":{"line":4,"col":5},"children":[{"name":"e1","kind":"Label","pos":{"line":4,"col":12},"children":[]},{"name":"e2","kind":"Label","pos":{"line":4,"col":23},"children":[]}]},{"name":"E","kind":"Exn","pos":{"line":2,"col":35},"children":[]},{"name":"t","kind":"Type","pos":{"line":2,"col":5},"children":[]},{"name":"t_of_sexp","kind":"Value","pos":{"line":2,"col":18},"children":[]}]]
